cob_sound/ros/src/sound.cpp

```diff
@@ -32,9 +35,11 @@ class SayAction
   {
     as_.start();
     srvServer_ = nh_.advertiseService("say", &SayAction::service_cb, this);
+    srvServer_ = nh_.advertiseService("play", &SayAction::service_cb_play, this);
```

Comment:
can the same ServiceServer advertise two services?
or do we need two ServiceServers - one for "say" and one for "play"?